src/components/SharedTables/AdvisoriesTable/index.tsx

```diff
@@ -96,16 +96,22 @@ export default function AdvisoriesTable({
           <Hide hide={isLoadingOrZeroCount}>
             <Thead>
               <Tr>
-                <Th />
-                {columnHeaders.map(({ name, width }, index) => (
-                  name === 'Name' || name === 'Synopsis' ?  
-                  <Th width={width as BaseCellProps['width']} key={index + name + '_header'}>
-                    {name}
-                  </Th> :
-                  <Th width={width as BaseCellProps['width']} key={index + name + '_header'} sort={sortParams(index)}>
-                  {name}
-                </Th>
-                ))}
+                <Th screenReaderText='empty' />
```

Comment:
i'm seeing this for this attribute: `Type '{ screenReaderText: string; }' is not assignable to type 'IntrinsicAttributes & Omit<ThProps, "ref"> & RefAttributes<HTMLTableHeaderCellElement>'.` was this intentional? or maybe i need to do an npm install?

Reply:
everything seems to run ok despite this error

Reply:
nevermind, npm install fixed that :)